File: js/js.translator/testData/box/jsQualifier/classes.kt
// EXPECTED_REACHABLE_NODES: 1282
// IGNORE_BACKEND: JS_IR
// MODULE: lib
// FILE: lib.kt
@file:JsQualifier("pkg")
external class C {
    fun o(): String

    class D {
        fun k(): String
    }
}

// MODULE: main(lib)
// FILE: main.kt

fun box() = C().o() + C.D().k()

File: js/js.translator/testData/box/jsQualifierOnDeclaration/classes.js
var pkg = function() {
    function C() {
    }
    C.prototype.o = function() {
        return "O";
    };
    function D() {
    }
    D.prototype.k = function() {
        return "K";
    };
    C.D = D;
    return {
        C: C
    };
}();